hoomd/hpmc/UpdaterRemoveDrift.h

```diff
@@ -66,8 +66,10 @@ class RemoveDriftUpdater : public Updater
                 {
                 unsigned int tag_i = h_tag.data[i];
                 // read in the current position and orientation
-                Scalar4 postype_i = h_postype.data[i];
-                vec3<Scalar> dr = vec3<Scalar>(postype_i) - vec3<Scalar>(h_r0.data[tag_i]) - origin;
+                vec3<Scalar> postype_i = vec3<Scalar>(h_postype.data[i]) - origin;
+		int3 tmp_image = make_int3(0, 0, 0);
```

Comment:
The formatting doesn't appear correct here. Are there tab characters? All indentation should be with spaces.